### PR TITLE
Adding license files to the repo

### PR DESCRIPTION
**Pull request by rareddy**
Signed-off-by: Ramesh Reddy <rareddy@jboss.org>

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+### Code reviews
+
+All submissions, including submissions by project members, need to be reviewed by at least two kas-installer committers before being merged.
+
+[GitHub Pull Request Review Process](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/about-pull-request-reviews) is followed for every pull request.
+
+### Coding Guidelines
+
+ * We decided to disallow `@author` tags in the Javadoc: they are hard to maintain
+ * Commits should be atomic and semantic. Please properly squash your pull requests before submitting them. Fixup commits can be used temporarily during the review process but things should be squashed at the end to have meaningful commits.
```
Comment (emmanuelbernard) [inline CONTRIBUTING.md:48]: Is the team following this practice? @k-wall @MikeEdgar

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+ We use merge commits so the GitHub Merge button cannot do that for us. If you don't know how to do that, just ask in your pull request, we will be happy to help!
```
Comment (emmanuelbernard) [inline CONTRIBUTING.md:49]: Same here, is that the practice followed?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+
+### Tests and documentation are not optional
```
Comment (emmanuelbernard) [inline CONTRIBUTING.md:52]: I don't think this project does that today.

**Review by emmanuelbernard — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 2 above)
Comment (MikeEdgar) [reply CONTRIBUTING.md:49]: Generally, yes. But we should disable squash merges if the team wants to enforce this. There are a few squash merges in history.

File: CONTRIBUTING.md (hunk 3 above)
Comment (MikeEdgar) [reply CONTRIBUTING.md:52]: Certainly tests are not used here, but typically we ask for newly-support env variables to be added to the template along with a short description.

File: CONTRIBUTING.md (hunk 1 above)
Comment (rareddy) [reply CONTRIBUTING.md]: removed the section as we can be flexible on this part in this project.

File: CONTRIBUTING.md (hunk 2 above)
Comment (rareddy) [reply CONTRIBUTING.md]: removed the section as we can be flexible on this part in this project.

**Review by MikeEdgar — APPROVED** [review]

**Pull request merged**